docs/installingNative.md

```diff
@@ -181,48 +193,76 @@ Most of these prerequisites can be installed using the package manager of your O
 
   You may need to restart the PostgreSQL server after changing the file above.
 
-- Run the test suite (Docker must be installed and running):
-
-  ```sh
-  make test
-  ```
+## Running
 
-- Run the linters:
+If you have your own [PrairieLearn course repository](./requestCourse/index.md), you will need to create the file `PrairieLearn/config.json` with the path of your local course repository. Additionally, if you need support for workspaces, you should provide a path to a directory into which temporary files will be saved when using the in-browser file editor. Here is a sample configuration:
 
-  ```sh
-  make lint # or lint-js for Javascript only, or lint-python for Python only
-  ```
+```json title="config.json"
+{
+  "courseDirs": ["/Users/mwest/git/pl-tam212", "exampleCourse"],
+  "filesRoot": "../filesRoot",
+  "workspaceHostHomeDirRoot": "/tmp/workspace",
+  "workspaceHomeDirRoot": "/tmp/workspace"
+}
+```
 
-- Create the file `PrairieLearn/config.json` with the path of your local course repository and with the path of a directory into which temporary files will be saved when using the in-browser file editor (edit both paths as needed):
+??? note "Advanced configuration"
 
-  ```json title="config.json"
-  {
-    "courseDirs": ["/Users/mwest/git/pl-tam212", "exampleCourse"],
-    "filesRoot": "../filesRoot",
-    "workspaceHostHomeDirRoot": "/tmp/workspace",
-    "workspaceHomeDirRoot": "/tmp/workspace"
-  }
-  ```
+    The full configuration can be found at [`apps/prairielearn/src/lib/config.ts`](https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/src/lib/config.ts).
```

Comment:
If this is under a separate heading, I'd be inclined to just link to this outside an admonition.

Reply:
Changed in [91dbf8a](https://github.com/PrairieLearn/PrairieLearn/pull/11957/commits/91dbf8a49875b10c0d4dd9c47a3f205cd08c1a9d).